go/vt/sqlparser/ast.go

```diff
@@ -1032,6 +1033,9 @@ const (
 // Format formats the node.
 func (node *ComparisonExpr) Format(buf *TrackedBuffer) {
 	buf.Myprintf("%v %s %v", node.Left, node.Operator, node.Right)
+	if node.Escape != nil {
+		buf.Myprintf(" escape %v", node.Escape)
+	}
```

Comment:
Added to WalkSubtree. Not sure what you mean by verifying that a nil expression doesn't crash it. I checked Walk and it does a nil-check. In terms of a nil expression in Format, it'd be a syntax error to have a ComparisonExpr without a left or right but with an escape.